demo/src/client/faro/initialize.ts

```diff
@@ -18,6 +18,7 @@ export function initializeFaro(): Faro {
     instrumentations: [
       ...getWebInstrumentations({
         captureConsole: true,
+        captureWebVitalsAttribution: true,
```

Comment:
Is there any problem with creating this option to facilitate the adoption of the instrumentation?